charts/gardener/gardenlet/templates/clusterrole-gardenlet.yaml

```
@@ -29,6 +29,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumes
+  verbs:
+  # This is currently necessary to migrate deprecated topology labels on PVs, ref: https://github.com/gardener/gardener/blob/e542bc4794473fcec1742ed751a2d420169c8326/cmd/gardenlet/app/migration.go#L27-L30.
+  # TODO(plkokanov): It can potentially be dropped once the requiring code is removed.
```

From https://github.com/gardener/gardener/blob/13f21090be9a4188b7ce277f76c20fe43ec85085/cmd/gardenlet/app/migration.go#L40:

```suggestion
  # TODO(plkokanov): Remove this RBAC rule when the requiring code is removed (when Kubernetes 1.27 support gets dropped).
```